src/code/z_kaleido_scope_call.c

```diff
@@ -60,6 +60,8 @@ void KaleidoScopeCall_Update(PlayState* play) {
     if (IS_PAUSED(&play->pauseCtx)) {
         if (pauseCtx->state == PAUSE_STATE_WAIT_LETTERBOX) {
             if (Letterbox_GetSize() == 0) {
+                // @note: line by line matches with the PAUSE_STATE_GAME_OVER_REQUEST below
+                // likely was an inlined function or a macro; or a copy-and-paste
```

Comment:
The compiler used (IDO) does not inline functions
It's probably just a copypaste, which is common enough I wouldn't comment on it at all

Reply:
got it, thanks. i've removed the comments then